<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Publications | Martina  Iannacito</title>
    <meta name="author" content="Martina  Iannacito">
    <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
">
    <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website">


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light">

    

    <!-- Styles -->
    
    <link rel="shortcut icon" href="/assets/img/cube.png">
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://martinaiannacito.github.io/publications/">

    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark">

    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Martina¬†</span>Iannacito</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>
          <!-- About
          <li class="toggle-container">
            <button id="home-toggle" title="Home" onclick="location.href='/'" value="Go to Google">
              <i class="fas fa-house-user"></i></button>
          </li>-->
          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">
              
              <li class="toggle-container">
                <button id="home-toggle" title="Home" style="visibility: hidden;">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </button>
              
              <button id="home-toggle" title="Home" onclick="location.href='/'" value="Go to Google">
                  <i class="fas fa-house-user"></i></button>
                </li>
                
              <!-- About               
              <li class="nav-item ">
                <a class="fas fa-house-user" href="/"></a>
              </li>-->
              

              <!-- Other pages -->
              <li class="nav-item active">
                <a class="nav-link" href="/publications/">Publications<span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/talks/">Talks</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/cv/">Resum√©</a>
              </li>
              <!-- Toogle theme mode -->
              <li class="toggle-container">
                <button id="light-toggle" title="Change theme">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </button>
              <!--
              <button id="home-toggle" title="Home" onclick="location.href='/'" value="Go to Google">
                  <i class="fas fa-house-user"></i></button>-->
                </li>
            </ul>
          </div>
        </div>
      </nav>

      <!-- Scrolling Progress Bar -->
      <progress id="progress" value="0">
        <div class="progress-container">
          <span class="progress-bar"></span>
        </div>
      </progress>
    </header>


    <!-- Content -->
    <div class="container mt-5">
      
        <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">Publications</h1>
            <p class="post-description"></p>
          </header>

          <article>
            <!-- _pages/publications.md -->
<div class="publications">

<h1 class="post-title">Preprints</h1>
  
  
  
    <h2 class="year">2025</h2>
    <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">arXiv</abbr></div>

        <!-- Entry bib key -->
        <div id="palitta2025SsCG" class="col-sm-8">
        <!-- Title -->
        <div class="title">A subspace-conjugate gradient method for linear matrix equations</div>
        <!-- Author -->
        <div class="author">
        

        Davide Palitta,¬†<em>Martina Iannacito</em>,¬†and¬†Valeria Simoncini</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          Jan 2025
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://arxiv.org/pdf/2501.02938" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The efficient solution of large-scale multiterm linear matrix equations is a challenging task in numerical linear algebra, and it is a largely open problem. We propose a new iterative scheme for symmetric and positive definite operators, significantly advancing methods such as truncated matrix-oriented Conjugate Gradients (CG). The new algorithm capitalizes on the low-rank matrix format of its iterates by fully exploiting the subspace information of the factors as iterations proceed. The approach implicitly relies on orthogonality conditions imposed over much larger subspaces than in CG, unveiling insightful connections with subspace projection methods. The new method is also equipped with memory-saving strategies. In particular, we show that for a given matrix Y, the action L(Y) in low rank format may not be evaluated exactly due to memory constraints. This problem is often underestimated, though it will eventually produce Out-of-Memory breakdowns for a sufficiently large number of terms. We propose an ad-hoc randomized range-finding strategy that appears to fully resolve this shortcoming. Experimental results with typical application problems illustrate the potential of our approach over various methods developed in the recent literature.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@techreport</span><span class="p">{</span><span class="nl">palitta2025SsCG</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A subspace-conjugate gradient method for linear matrix equations}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Palitta, Davide and Iannacito, Martina and Simoncini, Valeria}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{2501.02938}</span><span class="p">,</span>
  <span class="na">archiveprefix</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">primaryclass</span> <span class="p">=</span> <span class="s">{math.NA}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{report}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>
  

  
  
  
    <h2 class="year">2022</h2>
    <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">HAL</abbr></div>

        <!-- Entry bib key -->
        <div id="Coulaud2022b" class="col-sm-8">
        <!-- Title -->
        <div class="title">On some orthogonalization schemes in Tensor Train format</div>
        <!-- Author -->
        <div class="author">
        

        Olivier Coulaud,¬†Luc Giraud,¬†and¬†<em>Martina Iannacito</em>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          Nov 2022
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://hal.science/hal-03850387" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>In the framework of tensor spaces, we consider orthogonalization kernels to generate an orthogonal basis of a tensor subspace from a set of linearly independent tensors. In particular, we investigate numerically the loss of orthogonality of six orthogonalization methods, namely Classical and Modified Gram-Schmidt with (CGS2, MGS2) and without (CGS, MGS) re-orthogonalization, the Gram approach, and the Householder transformation. To tackle the curse of dimensionality, we represent tensor with low rank approximation using the Tensor Train (TT) formalism, and we introduce recompression steps in the standard algorithm outline through the TT-rounding method at a prescribed accuracy. After describing the algorithm structure and properties, we illustrate numerically that the theoretical bounds for the loss of orthogonality in the classical matrix computation round-off analysis results are maintained, with the unit round-off replaced by the TT-rounding accuracy. The computational analysis for each orthogonalization kernel in terms of the memory requirement and the computational complexity measured as a function of the number of TT-rounding, which happens to be the computational most expensive operation, completes the study.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@techreport</span><span class="p">{</span><span class="nl">Coulaud2022b</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{On some orthogonalization schemes in Tensor Train format}}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Coulaud, Olivier and Giraud, Luc and Iannacito, Martina}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{hal.archives-ouvertes.fr/hal-03850387}</span><span class="p">,</span>
  <span class="na">type</span> <span class="p">=</span> <span class="s">{Research Report}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{{RR-9491}}</span><span class="p">,</span>
  <span class="na">institution</span> <span class="p">=</span> <span class="s">{Inria center at the University of Bordeaux}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{report}</span><span class="p">,</span>
  <span class="na">hal_id</span> <span class="p">=</span> <span class="s">{hal-03850387}</span><span class="p">,</span>
  <span class="na">hal_version</span> <span class="p">=</span> <span class="s">{v2}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">HAL</abbr></div>

        <!-- Entry bib key -->
        <div id="Agullo2022" class="col-sm-8">
        <!-- Title -->
        <div class="title">The backward stable variants of GMRES in variable accuracy</div>
        <!-- Author -->
        <div class="author">
        

        Emmanuel Agullo,¬†Olivier Coulaud,¬†Luc Giraud, and
          <span class="more-authors" title="click to view 3 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '3 more authors' ? 'Martina Iannacito, Gilles Marait, Nick Schenkels' : '3 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '99');
              ">3 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          Sep 2022
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://hal.science/hal-03776837" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>In the context where the representation of the data is decoupled from the arithmetic used to process them, we investigate the backward stability of two backward-stable implementations of the GMRES method, namely the so-called Modified Gram-Schmidt (MGS) and the Householder variants. Considering data may be compressed to alleviate the memory footprint, we are interested in the situation where the leading part of the rounding error is related to the data representation. When the data representation of vectors introduces componentwise perturbations, we show that the existing backward stability analyses of MGS-GMRES and Householder-GMRES still apply. We illustrate this backward stability property in a practical context where an agnostic lossy compressor is employed and enables the reduction of the memory requirement to store the orthonormal Arnoldi basis or the Householder reflectors. Although technical arguments of the theoretical backward stability proofs do not readily apply to the situation where only the normwise relative perturbations of the vector storage can be controlled, we show experimentally that the backward stability is maintained; that is, the attainable normwise backward error is of the same order as the normwise perturbations induced by the data storage. We illustrate it with numerical experiments in two practical different contexts. The first one corresponds to the use of an agnostic compressor where vector compression is controlled normwise. The second one arises in the solution of tensor linear systems, where low-rank tensor approximations based on Tensor-Train is considered to tackle the curse of dimensionality.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@techreport</span><span class="p">{</span><span class="nl">Agullo2022</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{The backward stable variants of GMRES in variable accuracy}}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Agullo, Emmanuel and Coulaud, Olivier and Giraud, Luc and Iannacito, Martina and Marait, Gilles and Schenkels, Nick}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://hal.science/hal-03776837}</span><span class="p">,</span>
  <span class="na">type</span> <span class="p">=</span> <span class="s">{Research Report}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{RR-9483}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1-77}</span><span class="p">,</span>
  <span class="na">institution</span> <span class="p">=</span> <span class="s">{Inria center at the University of Bordeaux}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">sep</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{report}</span><span class="p">,</span>
  <span class="na">hal_id</span> <span class="p">=</span> <span class="s">{hal-03776837}</span><span class="p">,</span>
  <span class="na">hal_version</span> <span class="p">=</span> <span class="s">{v3}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">HAL</abbr></div>

        <!-- Entry bib key -->
        <div id="Coulaud2022a" class="col-sm-8">
        <!-- Title -->
        <div class="title">A robust GMRES algorithm in Tensor Train format</div>
        <!-- Author -->
        <div class="author">
        

        Olivier Coulaud,¬†Luc Giraud,¬†and¬†<em>Martina Iannacito</em>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          Sep 2022
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://inria.hal.science/hal-03776529v4/file/RR-9484.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We consider the solution of linear systems with tensor product structure using a GMRES algorithm. In order to cope with the computational complexity in large dimension both in terms of floating point operations and memory requirement, our algorithm is based on low-rank tensor representation, namely the Tensor Train format. In a backward error analysis framework, we show how the tensor approximation affects the accuracy of the computed solution. With the bacwkward perspective, we investigate the situations where the (ùëë+1)-dimensional problem to be solved results from the concatenation of a sequence of ùëë-dimensional problems (like parametric linear operator or parametric right-hand side problems), we provide backward error bounds to relate the accuracy of the (ùëë+1)-dimensional computed solution with the numerical quality of the sequence of ùëë-dimensional solutions that can be extracted form it. This enables to prescribe convergence threshold when solving the (ùëë+1)-dimensional problem that ensures the numerical quality of the ùëë-dimensional solutions that will be extracted from the (ùëë+1)-dimensional computed solution once the solver has converged. The above mentioned features are illustrated on a set of academic examples of varying dimensions and sizes.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@techreport</span><span class="p">{</span><span class="nl">Coulaud2022a</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{A robust GMRES algorithm in Tensor Train format}}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Coulaud, Olivier and Giraud, Luc and Iannacito, Martina}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://hal.science/hal-03776529v4}</span><span class="p">,</span>
  <span class="na">type</span> <span class="p">=</span> <span class="s">{Research Report}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{RR-9484}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1-48}</span><span class="p">,</span>
  <span class="na">institution</span> <span class="p">=</span> <span class="s">{Inria center at the University of Bordeaux}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">sep</span><span class="p">,</span>
  <span class="na">hal_id</span> <span class="p">=</span> <span class="s">{hal-03776529}</span><span class="p">,</span>
  <span class="na">hal_version</span> <span class="p">=</span> <span class="s">{v4}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{report}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>
  

  
  
  
    <h2 class="year">2021</h2>
    <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">HAL</abbr></div>

        <!-- Entry bib key -->
        <div id="Coulaud2021" class="col-sm-8">
        <!-- Title -->
        <div class="title">Extension of Correspondence Analysis to multiway data-sets through High Order SVD: a geometric framework</div>
        <!-- Author -->
        <div class="author">
        

        Olivier Coulaud,¬†Alain Franc,¬†and¬†<em>Martina Iannacito</em>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          Nov 2021
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://hal.science/hal-03418404" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>This paper presents an extension of Correspondence Analysis (CA) to tensors through High Order Singular Value Decomposition (HOSVD) from a geometric viewpoint. Correspondence analysis is a well-known tool, developed from principal component analysis, for studying contingency tables. Different algebraic extensions of CA to multi-way tables have been proposed over the years, nevertheless neglecting its geometric meaning. Relying on the Tucker model and the HOSVD, we propose a direct way to associate with each tensor mode a point cloud. We prove that the point clouds are related to each other. Specifically using the CA metrics we show that the barycentric relation is still true in the tensor framework. Finally two data sets are used to underline the advantages and the drawbacks of our strategy with respect to the classical matrix approaches.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@techreport</span><span class="p">{</span><span class="nl">Coulaud2021</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Extension of Correspondence Analysis to multiway data-sets through High Order SVD: a geometric framework}}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Coulaud, Olivier and Franc, Alain and Iannacito, Martina}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://hal.science/hal-03418404}</span><span class="p">,</span>
  <span class="na">type</span> <span class="p">=</span> <span class="s">{Research Report}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{RR-9429}</span><span class="p">,</span>
  <span class="na">institution</span> <span class="p">=</span> <span class="s">{Inria center at the University of Bordeaux; INRAE}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{report}</span><span class="p">,</span>
  <span class="na">hal_id</span> <span class="p">=</span> <span class="s">{hal-03418404}</span><span class="p">,</span>
  <span class="na">hal_version</span> <span class="p">=</span> <span class="s">{v1}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>
  

  
  
  


<h1 class="post-title">Peer-reviewed journal</h1>
  
  
  
  
  
    <h2 class="year">2025</h2>
    <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">ETNA</abbr></div>

        <!-- Entry bib key -->
        <div id="Coulaud2025Note" class="col-sm-8">
        <!-- Title -->
        <div class="title">A note on TT-GMRES for the solution of parametric linear systems</div>
        <!-- Author -->
        <div class="author">
        

        Olivier Coulaud,¬†Luc Giraud,¬†and¬†<em>Martina Iannacito</em>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Electronic Transactions on Numerical Analysis</em>, Jan 2025
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.1553/etna_vol62s163" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We study the solution of linear systems with tensor product structure using the Generalized Minimal RESidual (GMRES) algorithm. To manage the computational complexity of high-dimensional problems, our approach relies on low-rank tensor representation, focusing specifically on the Tensor Train format. We implement and experimentally study the TT-GMRES algorithm. Our analysis bridges the heuristic methods proposed for TT-GMRES by Dolgov [Russian J. Numer. Anal. Math. Modelling, 28 (2013), pp. 149-172] and the theoretical framework of inexact GMRES by Simoncini and Szyld [SIAM J. Sci. Comput. 25 (2003), pp. 454-477]. This approach is particularly relevant in a scenario where a (d+1)-dimensional problem arises from concatenating a sequence of d-dimensional problems, as in the case of a parametric linear operator or parametric right-hand-side formulation. Thus, we provide backward error bounds that link the accuracy of the computed (d+1)-dimensional solution to the numerical quality of the extracted d-dimensional solutions. This facilitates the prescription of a convergence threshold ensuring that the d-dimensional solutions extracted from the (d+1)-dimensional result have the desired accuracy once the solver converges. We illustrate these results with academic examples across varying dimensions and sizes. Our experiments indicate that TT-GMRES retains the theoretical rounding-error properties observed in matrix-based GMRES.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Coulaud2025Note</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A note on TT-GMRES for the solution of parametric linear systems}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Coulaud, Olivier and Giraud, Luc and Iannacito, Martina}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Electronic Transactions on Numerical Analysis}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Kent State University Library}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{62}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{163-187}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1553/etna_vol62s163}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{intjour}</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{journal}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>
  

  
  
  
  
  

  
  
  
  
  
    <h2 class="year">2021</h2>
    <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">BUMI</abbr></div>

        <!-- Entry bib key -->
        <div id="Bernardi2021" class="col-sm-8">
        <!-- Title -->
        <div class="title">High order singular value decomposition for plant diversity estimation</div>
        <!-- Author -->
        <div class="author">
        

        Alessandra Bernardi,¬†<em>Martina Iannacito</em>,¬†and¬†Duccio Rocchini</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Bollettino dell‚ÄôUnione Matematica Italiana</em>, Dec 2021
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://link.springer.com/article/10.1007/s40574-021-00300-w" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We propose a new method to estimate plant diversity with R√©nyi and Rao indexes through the so called High Order Singular Value Decomposition (HOSVD) of tensors. Starting from NASA multi-spectral images we evaluate diversity and we compare original diversity estimates with those realized via the HOSVD compression methods for big data. Our strategy turns out to be extremely powerful in terms of memory storage and precision of the outcome. The obtained results are so promising that we can support the efficiency of our method in the ecological framework.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Bernardi2021</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Bernardi, Alessandra and Iannacito, Martina and Rocchini, Duccio}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{High order singular value decomposition for plant diversity estimation}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Bollettino dell'Unione Matematica Italiana}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">dec</span><span class="p">,</span>
  <span class="na">day</span> <span class="p">=</span> <span class="s">{01}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{557-591}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2198-2759}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s40574-021-00300-w}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{natjour}</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{journal}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">Comm. Eco.</abbr></div>

        <!-- Entry bib key -->
        <div id="Thouverai2021" class="col-sm-8">
        <!-- Title -->
        <div class="title">Measuring diversity from space: a global view of the free and open source rasterdiv R package under a coding perspective</div>
        <!-- Author -->
        <div class="author">
        

        Elisa Thouverai,¬†Matteo Marcantonio,¬†Giovanni Bacaro, and
          <span class="more-authors" title="click to view 7 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '7 more authors' ? 'Daniele Da Re, Martina Iannacito, Elisa Marchetto, Carlo Ricotta, Clara Tattoni, Saverio Vicario, Duccio Rocchini' : '7 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '99');
              ">7 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Community Ecology</em>, Apr 2021
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://doi.org/10.1007/s42974-021-00042-x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The variation of species diversity over space and time has been widely recognised as a key challenge in ecology. However, measuring species diversity over large areas might be difficult for logistic reasons related to both time and cost savings for sampling, as well as accessibility of remote ecosystems. In this paper, we present a new R package - rasterdiv - to calculate diversity indices based on remotely sensed data, by discussing the theory behind the developed algorithms. Obviously, measures of diversity from space should not be viewed as a replacement of in situ data on biological diversity, but they are rather complementary to existing data and approaches. In practice, they integrate available information of Earth surface properties, including aspects of functional (structural, biophysical and biochemical), taxonomic, phylogenetic and genetic diversity. Making use of the rasterdiv package can result useful in making multiple calculations based on reproducible open source algorithms, robustly rooted in Information Theory.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Thouverai2021</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Thouverai, Elisa and Marcantonio, Matteo and Bacaro, Giovanni and Re, Daniele Da and Iannacito, Martina and Marchetto, Elisa and Ricotta, Carlo and Tattoni, Clara and Vicario, Saverio and Rocchini, Duccio}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Measuring diversity from space: a global view of the free and open source rasterdiv R package under a coding perspective}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Community Ecology}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
  <span class="na">day</span> <span class="p">=</span> <span class="s">{01}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{22}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1-11}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1588-2756}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s42974-021-00042-x}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{intjour}</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{journal}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">Glob. Ecol.</abbr></div>

        <!-- Entry bib key -->
        <div id="Rocchini2021b" class="col-sm-8">
        <!-- Title -->
        <div class="title">From zero to infinity: Minimum to maximum diversity of the planet by spatio-parametric Rao‚Äôs quadratic entropy</div>
        <!-- Author -->
        <div class="author">
        

        Duccio Rocchini,¬†Matteo Marcantonio,¬†Daniele Da Re, and
          <span class="more-authors" title="click to view 20 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '20 more authors' ? 'Giovanni Bacaro, Enrico Feoli, Giles M. Foody, Reinhard Furrer, Ryan J. Harrigan, David Kleijn, Martina Iannacito, Jonathan Lenoir, Meixi Lin, Marco Malavasi, Elisa Marchetto, Rachel S. Meyer, V√≠tƒõzslav Moudry, Fabian D. Schneider, Petra ≈†√≠mov√°, Andrew H. Thornhill, Elisa Thouverai, Saverio Vicario, Robert K. Wayne, Carlo Ricotta' : '20 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '99');
              ">20 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Global Ecology and Biogeography</em>, Mar 2021
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://hal.science/hal-03363010" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The majority of work done to gather information on the Earth‚Äôs biodiversity has been carried out using in-situ data, with known issues related to epistemology (e.g., species determination and taxonomy), spatial uncertainty, logistics (time and costs), among others. An alternative way to gather information about spatial ecosystem variability is the use of satellite remote sensing. It works as a powerful tool for attaining rapid and standardized information. Several metrics used to calculate remotely sensed diversity of ecosystems are based on Shannon‚Äôs information theory, namely on the differences in relative abundance of pixel reflectances in a certain area. Additional metrics like the Rao‚Äôs quadratic entropy allow the use of spectral distance beside abundance, but they are point descriptors of diversity, that is they can account only for a part of the whole diversity continuum. The aim of this paper is thus to generalize the Rao‚Äôs quadratic entropy by proposing its parameterization for the first time. Innovation The parametric Rao‚Äôs quadratic entropy, coded in R, (a) allows the representation of the whole continuum of potential diversity indices in one formula, and (b) starting from the Rao‚Äôs quadratic entropy, allows the explicit use of distances among pixel reflectance values, together with relative abundances. Main conclusions The proposed unifying measure is an integration between abundance- and distance-based algorithms to map the continuum of diversity given a satellite image at any spatial scale. Being part of the rasterdiv R package, the proposed method is expected to ensure high robustness and reproducibility.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Rocchini2021b</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Rocchini, Duccio and Marcantonio, Matteo and Da Re, Daniele and Bacaro, Giovanni and Feoli, Enrico and Foody, Giles M. and Furrer, Reinhard and Harrigan, Ryan J. and Kleijn, David and Iannacito, Martina and Lenoir, Jonathan and Lin, Meixi and Malavasi, Marco and Marchetto, Elisa and Meyer, Rachel S. and Moudry, V√≠tƒõzslav and Schneider, Fabian D. and ≈†√≠mov√°, Petra and Thornhill, Andrew H. and Thouverai, Elisa and Vicario, Saverio and Wayne, Robert K. and Ricotta, Carlo}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{From zero to infinity: Minimum to maximum diversity of the planet by spatio-parametric Rao‚Äôs quadratic entropy}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Global Ecology and Biogeography}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{30}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1153-1162}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{intjour}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1111/geb.13270}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{onlinelibrary.wiley.com/doi/abs/10.1111/geb.13270}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">mar</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{journal}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">MEE</abbr></div>

        <!-- Entry bib key -->
        <div id="Rocchini2021a" class="col-sm-8">
        <!-- Title -->
        <div class="title">rasterdiv‚ÄîAn Information Theory tailored R package for measuring ecosystem heterogeneity from space: To the origin and back</div>
        <!-- Author -->
        <div class="author">
        

        Duccio Rocchini,¬†Elisa Thouverai,¬†Matteo Marcantonio, and
          <span class="more-authors" title="click to view 32 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '32 more authors' ? 'Martina Iannacito, Daniele Da Re, Michele Torresani, Giovanni Bacaro, Manuele Bazzichetto, Alessandra Bernardi, Giles M. Foody, Reinhard Furrer, David Kleijn, Stefano Larsen, Jonathan Lenoir, Marco Malavasi, Elisa Marchetto, Filippo Messori, Alessandro Monkeywordhi, V√≠tƒõzslav Moudr√Ω, Babak Naimi, Carlo Ricotta, Micol Rossini, Francesco Santi, Maria J. Santos, Michael E. Schaepman, Fabian D. Schneider, Leila Schuh, Sonia Silvestri, Petra ≈ú√≠mov√°, Andrew K. Skidmore, Clara Tattoni, Enrico Tordoni, Saverio Vicario, Piero Zannini, Martin Wegmann' : '32 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '99');
              ">32 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Methods in Ecology and Evolution</em>, Feb 2021
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://besjournals.onlinelibrary.wiley.com/doi/epdf/10.1111/2041-210X.13583" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Ecosystem heterogeneity has been widely recognized as a key ecological indicator of several ecological functions, diversity patterns and change, metapopulation dynamics, population connectivity or gene flow. In this paper, we present a new R package‚Äîrasterdiv‚Äîto calculate heterogeneity indices based on remotely sensed data. We also provide an ecological application at the landscape scale and demonstrate its power in revealing potentially hidden heterogeneity patterns. The rasterdiv package allows calculating multiple indices, robustly rooted in Information Theory, and based on reproducible open-source algorithms.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Rocchini2021a</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Rocchini, Duccio and Thouverai, Elisa and Marcantonio, Matteo and Iannacito, Martina and Da Re, Daniele and Torresani, Michele and Bacaro, Giovanni and Bazzichetto, Manuele and Bernardi, Alessandra and Foody, Giles M. and Furrer, Reinhard and Kleijn, David and Larsen, Stefano and Lenoir, Jonathan and Malavasi, Marco and Marchetto, Elisa and Messori, Filippo and Monkeywordhi, Alessandro and Moudr√Ω, V√≠tƒõzslav and Naimi, Babak and Ricotta, Carlo and Rossini, Micol and Santi, Francesco and Santos, Maria J. and Schaepman, Michael E. and Schneider, Fabian D. and Schuh, Leila and Silvestri, Sonia and ≈ú√≠mov√°, Petra and Skidmore, Andrew K. and Tattoni, Clara and Tordoni, Enrico and Vicario, Saverio and Zannini, Piero and Wegmann, Martin}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{rasterdiv‚ÄîAn Information Theory tailored R package for measuring ecosystem heterogeneity from space: To the origin and back}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Methods in Ecology and Evolution}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1093-1102}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{intjour}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1111/2041-210X.13583}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13583}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">tag</span> <span class="p">=</span> <span class="s">{journal}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li>
</ol>
  

  
  
  
  
  



<h1 class="post-title">Other works</h1>
  
  
  

  
  
  
    <h2 class="year">2022</h2>
    <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">thesis</abbr></div>

        <!-- Entry bib key -->
        <div id="Iannacito2022" class="col-sm-8">
        <!-- Title -->
        <div class="title">Numerical linear algebra and data analysis in large dimensions using tensor format</div>
        <!-- Author -->
        <div class="author">
        

        <em>Martina Iannacito</em>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Inria center at the University of Bordeaux, France</em>, Dec 2022
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="https://theses.hal.science/tel-03952711v1/document" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a>
            <a href="/assets/pdf/Defence_09-12-22.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>This work aims to establish which theoretical properties of classical linear algebra techniques, developed in two different contexts, namely numerical linear algebra and data analysis, are maintained and which are lost when extended to low-rank tensors. In the numerical linear algebra part, we experimentally study the effects of rounding errors on an iterative solver (Generalised Minimal RESidual) and several orthogonalization methods (Classical and Modified Gram-Schmidt and Householder transformation, among others) when extended to tensors by the Tensor Train (TT) formalism. In all the considered algorithms, we introduce additional rounding steps, with the TT-rounding compression algorithm, to cope with memory constraints, which are always crucial when dealing with tensors. Our experimental tests suggest that for these algorithms, the classical bounds due to the propagation of rounding errors remain valid, replacing the accuracy of the arithmetic with that of the TT-rounding algorithm. In the data analysis part, we geometrically study the generalization of correspondence analysis to multi-way tables by the Tucker tensor decomposition technique, thus contributing to the understanding of Multi-Way Correspondence Analysis (MWCA). Examples of MWCA applied to datasets complement the theoretical results. In particular, we perform the MWCA on an original ecological dataset provided to us in the framework of the Malabar project of the BioGeCo (INRAE) and Pleiade (Inria) teams.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@phdthesis</span><span class="p">{</span><span class="nl">Iannacito2022</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Iannacito, Martina}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">dec</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Numerical linear algebra and data analysis in large dimensions using tensor format}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{theses.fr/s349733}</span><span class="p">,</span>
  <span class="na">school</span> <span class="p">=</span> <span class="s">{Inria center at the University of Bordeaux, France}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{thesis}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>
  

  
  
  

  
  
  
    <h2 class="year">2019</h2>
    <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">thesis</abbr></div>

        <!-- Entry bib key -->
        <div id="Iannacito2019" class="col-sm-8">
        <!-- Title -->
        <div class="title">HOSVD FOR MULTISPECTRAL IMAGES. A numerical approach to the plant biodiversity estimate.</div>
        <!-- Author -->
        <div class="author">
        

        <em>Martina Iannacito</em>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>University of Trento, Italy</em>, Jul 2019
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
            <a href="/assets/pdf/Master_thesis.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
            <a href="/assets/pdf/Defence_17-07-19.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Remote sensing has already proved its power in estimating plant biodiversity. Since plants reflect and absorb in a specific way sunlight, ecologists use multispectral photos to estimate plant biodiversity of a certain area through biodiversity indexes, e.g., R√©nyi‚Äôs and Rao‚Äôs index. Working with these big data is often expensive in terms of storage memory requirements. A possible solution is compressing multispectral data through tensor techniques, as High Order Singular Value Decomposition. We apply two recent variants of HOSVD to spectral images and we compute biodiversity indexes over the compressed data generated. For the R√©nyi index compressing data is extremely advantageous, since the information loss in estimating biodiversity is very low. For Rao‚Äôs index the results are not as satisfactory, even if they are promising. Using approximatively 15% of the original data information, in the first case we estimate an average error per pixel between 5.5% and 7.5%, while in the second case the average error per pixel is between 17% and 19%.</p>
          </div>
<!-- Hidden bibtex block -->
          <div class="bibtex hidden">
            <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@mastersthesis</span><span class="p">{</span><span class="nl">Iannacito2019</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Iannacito, Martina}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jul</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{HOSVD FOR MULTISPECTRAL IMAGES. A} numerical approach to the plant biodiversity estimate.}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{webapps.unitn.it/Biblioteca/it/Web/RichiestaConsultazioneTesi/365719}</span><span class="p">,</span>
  <span class="na">school</span> <span class="p">=</span> <span class="s">{University of Trento, Italy}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{thesis}</span><span class="p">,</span>
  <span class="na">type</span> <span class="p">=</span> <span class="s">{thesis}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure>
          </div>
        </div>
      </div>
</li></ol>
  

</div>

          </article>

        </div>

      
    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        ¬© Copyright 2025 Martina  Iannacito. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>.

      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js"></script>

  <!-- Load Common JS -->
  <script defer src="/assets/js/common.js"></script>
  <script defer src="/assets/js/copy_code.js" type="text/javascript"></script>

    

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
    

<!-- Scrolling Progress Bar -->
<script type="text/javascript">
  /*
   * This JavaScript code has been adapted from the article 
   * https://css-tricks.com/reading-position-indicator/ authored by Pankaj Parashar, 
   * published on the website https://css-tricks.com on the 7th of May, 2014.
   * Couple of changes were made to the original code to make it compatible 
   * with the `al-foio` theme.
   */
  const progressBar = $("#progress");
  /*
   * We set up the bar after all elements are done loading.
   * In some cases, if the images in the page are larger than the intended
   * size they'll have on the page, they'll be resized via CSS to accomodate
   * the desired size. This mistake, however, breaks the computations as the
   * scroll size is computed as soon as the elements finish loading.
   * To account for this, a minimal delay was introduced before computing the
   * values.
   */
  window.onload = function () {
    setTimeout(progressBarSetup, 50);
  };
  /*
   * We set up the bar according to the browser.
   * If the browser supports the progress element we use that.
   * Otherwise, we resize the bar thru CSS styling
   */
  function progressBarSetup() {
    if ("max" in document.createElement("progress")) {
      initializeProgressElement();
      $(document).on("scroll", function() {
        progressBar.attr({ value: getCurrentScrollPosition() });
      });
      $(window).on("resize", initializeProgressElement);
    } else {
      resizeProgressBar();
      $(document).on("scroll", resizeProgressBar);
      $(window).on("resize", resizeProgressBar);
    }
  }
  /*
   * The vertical scroll position is the same as the number of pixels that
   * are hidden from view above the scrollable area. Thus, a value > 0 is
   * how much the user has scrolled from the top
   */
  function getCurrentScrollPosition() {
    return $(window).scrollTop();
  }

  function initializeProgressElement() {
    let navbarHeight = $("#navbar").outerHeight(true);
    $("body").css({ "padding-top": navbarHeight });
    $("progress-container").css({ "padding-top": navbarHeight });
    progressBar.css({ top: navbarHeight });
    progressBar.attr({
      max: getDistanceToScroll(),
      value: getCurrentScrollPosition(),
    });
  }
  /*
   * The offset between the html document height and the browser viewport
   * height will be greater than zero if vertical scroll is possible.
   * This is the distance the user can scroll
   */
  function getDistanceToScroll() {
    return $(document).height() - $(window).height();
  }

  function resizeProgressBar() {
    progressBar.css({ width: getWidthPercentage() + "%" });
  }
  // The scroll ratio equals the percentage to resize the bar
  function getWidthPercentage() {
    return (getCurrentScrollPosition() / getDistanceToScroll()) * 100;
  }
</script>

  </body>
</html>
